src/classes/fflib_InOrder.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>37.0</apiVersion>
```

Comment:
i thought it was 37 the API version

https://github.com/financialforcedev/fflib-apex-mocks/pull/24